### PR TITLE
Add some more ImageBitmap tests.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/22778)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/328996240)

File: 2dcontext/imagebitmap/createImageBitmap-origin.sub.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+function makeImage() {
+  return new Promise((resolve, reject) => {
+    const image = new Image();
+    image.onload = () => resolve(image);
+    image.onerror = reject;
+    image.src = crossOriginImageUrl;
+  });
+}
+
+const arguments = [
```
Comment (Ms2ger) [inline 2dcontext/imagebitmap/createImageBitmap-origin.sub.html]: @annevk @zcorpan if either of you have ideas of more cases that would be useful to test, please let me know.

Comment (annevk) [reply 2dcontext/imagebitmap/createImageBitmap-origin.sub.html]: Maybe canvas backed by a worker canvas and such?

**Review by annevk — APPROVED** [review]

**Comment by annevk** [comment]
Noted the many Firefox failures in https://bugzilla.mozilla.org/show_bug.cgi?id=1419392, FWIW.

**Pull request merged**